pebblo/app/models/models.py

```diff
@@ -167,26 +167,62 @@ class ReportModel(BaseModel):
     pebbloClientVersion: Optional[str]
 
 
-class AppListDetails(BaseModel):
+class LoaderAppListDetails(BaseModel):
     name: str
     topics: int
     entities: int
     owner: Optional[str]
     loadId: Optional[str]
 
 
-class AppModel(BaseModel):
+class LoaderAppModel(BaseModel):
     applicationsAtRiskCount: int
     findingsCount: int
     documentsWithFindingsCount: int
     dataSourceCount: int
-    appList: List[AppListDetails]
+    appList: List[LoaderAppListDetails]
     findings: list
     documentsWithFindings: list
     dataSource: list
     pebbloServerVersion: Optional[str]
 
 
+class RetrievalContext(BaseModel):
+    retrieved_from: str
+    doc: str
+    vector_db: str
+
+
+class RetrievalData(BaseModel):
+    context: List[RetrievalContext]
+    prompt: AiDataModel
+    response: AiDataModel
+    prompt_time: str
+    user: str
+
+
+class RetrievalAppListDetails(BaseModel):
+    name: str = ""
+    owner: str = ""
+    retrievals: list[RetrievalData] = []
+    active_users: list[str] = []
+    vector_dbs: list[str] = []
+
+
+class RetrievalAppModel(BaseModel):
+    appList: list = []
+    retrievals: list = []
+    activeUsers: list = []
```

Comment:
UI needs count of active users rather than all the details.

Reply:
We will be sending active_users details, so it can be used to show active users details in activeUsers Tab, its count can be taken out be len() method